pkg/sfu/downtrack.go

```diff
@@ -1216,15 +1230,17 @@ func (d *DownTrack) handleRTCP(bytes []byte) {
 
 func (d *DownTrack) SetConnected() {
 	if !d.connected.Swap(true) {
-		if d.bound.Load() && d.kind == webrtc.RTPCodecTypeVideo {
```

Comment:
If the downtrack is not bound when SetConnected is called, it will not send the PLI on connected because connected is swapped to true, move these code to `onBindAndConnected` and check it both at `OnBind` and `SetConnected`